crates/sui-core/src/authority_active.rs

```diff
@@ -259,6 +259,10 @@ impl<A> ActiveAuthority<A> {
         let entry = lock.entry(name).or_default();
         entry.can_initiate_contact_now()
     }
+
+    pub fn clone_node_sync_state(&self) -> Arc<NodeSyncState<A>> {
+        self.node_sync_state.clone()
+    }
```

Comment:
This method probably doesn't compile after https://github.com/MystenLabs/sui/pull/4825, but you shouldn't need it anymore either.